### PR TITLE
Moods foundation

### PR DESCRIPTION
**Pull request by konstantinmvelkov**
Closes https://trello.com/c/PK6teHyK

### CODE REVIEW

File: src/routes/mood.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const moodRouter = require('express').Router();
+
+moodRouter.post('/mood', (req: any, res: any) => {
```
Comment (RinseV) [inline src/routes/mood.ts:4]: Not sure what this post is for, ideally the POSTing of the mood data (to GameBus) would happen in the MoodParser's `post` function, but we'll see.

File: src/gb/models/moodModel.ts [hunk 2]
```diff
@@ -1 +1,5 @@
-// TODO
+export interface MoodModel {
```
Comment (RinseV) [inline src/gb/models/moodModel.ts:1]: This will probably have to be tweaked once the front-end implementation is finalized.

**Review by RinseV — APPROVED** [review]
Might need some changes later on depending on front-end

File: src/routes/mood.ts (hunk 1 above)
Comment (konstantinmvelkov) [reply src/routes/mood.ts:4]: This is the endpoint to which the front-end sends the entry

File: src/gb/models/moodModel.ts (hunk 2 above)
Comment (konstantinmvelkov) [reply src/gb/models/moodModel.ts:1]: Yes that is true, but we can open a new PR for that once it is done.

**Pull request merged**